Cargo.toml

```diff
@@ -39,6 +39,7 @@ features = ['unprefixed_malloc_on_supported_platforms']
 libc = "0.2"
 libffi = "4.0.0"
 libloading = "0.8"
+nix = { version = "0.30.1", features = ["mman", "ptrace", "signal"] }
```

Comment:
We don't need all these features for this PR, do we?

Reply:
Correct, oops. I'll change that